### ISSUE TITLE
Feature Request: support endian argument to createVariable

### ISSUE THREAD

**Issue by glenfletcher** [h5netcdf/h5netcdf#15]
You mention in the readme "We don't support the endian argument to createVariable. The h5py API does not appear to offer this feature."

h5py dose support endianness buts its a component of the dtype i.e. `dtype='<f8'` is little, while `dtype='>f8'` is big, you can use the `newbyteorder` function on a numpy dype object to set it like so

```
>>> np.dtype("<f, >f")
dtype([('f0', '<f4'), ('f1', '>f4')])
>>> np.dtype("<f, >f").newbyteorder('>')
dtype([('f0', '>f4'), ('f1', '>f4')])
>>> np.dtype("<f, >f").newbyteorder('<')
dtype([('f0', '<f4'), ('f1', '<f4')])
```

**Comment by shoyer** [comment]
Thanks for pointing this out. If anyone is interested in this feature, please make a pull request.

**Comment by kmuehlbauer** [comment]
@glenfletcher 5+ years is a long time, but maybe you are still looking for that feature. It's approaching in #124.

**Comment by kmuehlbauer** [comment]
Closed via #124
